docs/articles/documentation/using-testcafe/common-concepts/using-browsers.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: docs
+title: Using Browsers
+permalink: /documentation/using-testcafe/common-concepts/using-browsers.html
+---
+# Using Browsers
+
+TestCafe comes with a [built-in library](#built-in-library) and several [plugins](#browser-provider-plugins) that allow you to easily set up testing in different web browsers.
```

Comment:
`TestCafe comes with a [built-in library](#built-in-library)` I don't get it. We have just built-in support for popular browsers, remote browsers and specifying browser via path. `browser-natives` has nothing to do with it from end-user perspective, it's just an utility library that we use internally and that can be used by provider plugins. Also it has another name now